nextcloud_install_production.sh

```diff
@@ -2,6 +2,71 @@
 
 # T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
 
+# Variables:
+# Todo: make DOWNLOAD_DIR readable again; is disabled for now to fix travis
+SCRIPTS=/var/scripts
+#DOWNLOAD_DIR=/tmp/NC_VM
+
+# Download and validate Nextcloud VM
+if ! [ -d $SCRIPTS ]
```

Comment:
@enoch85 is the /var/scripts directory always non-existant on a fresh ubuntu 20.04 installation?